dna/modify.go

```diff
@@ -1,11 +1,17 @@
 package dna
 
 import (
+	"errors"
 	"fmt"
 	"github.com/vertgenlab/gonomics/common"
 	"log"
 )
 
+var (
+	ErrInvalidPosition          = errors.New("Input position is invalid")
+	ErrExceedCapNewSliceCreated = errors.New("Capacity of input slice was exceeded, output slice was created at a new memory address")
+)
+
```

Comment:
I actually have a question about the code starting at line 15. For the if...else if...else statement; why bother if the b == Gap or b == Dot returning "b" when the else statement will also just return "b"? Wouldnt this code do exactly the same thing? :
if b > N {
     return b - 5
} else {
     return b
}

Reply:
In the definition of Base in dna/dna.go each base is a byte with a numerical value A=0, C=1... N=4... Gap=10, Dot=11. Since the numerical value assigned to Gap and Dot is greater than N, they would be caught by the first else statement and be changed to "a" and "c" respectively. To avoid this, they are handled in a separate if statement

Reply:
So here we only want to return b-5 if b is greater than 4 (N) but is not equal to 10 (Gap) or 11 (Dot). And I assume you use "N", "Gap", and "Dot" instead of 4, 10, 11, since values are arbitrary and can be changed?